app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def markdown(text)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+    return markdown.render(text).html_safe
```

Comment:
No need to use a return here, as ruby returns the last thing in a method automatically.